cpp/daal/src/algorithms/algorithm_hyperparameter.cpp

```diff
@@ -65,11 +66,17 @@ struct HyperparameterImpl : public HyperparameterBaseImpl
     }
 
 protected:
+    
+    #ifdef __ARM_ARCH
+        #define CPU_ARG sve
```

Comment:
This is a default CPU, or minimum CPU? The name should be more specific than just CPU_ARG. @napetrov any suggestions for a good name?